components/brave_wallet_ui/components/desktop/popup-modals/style.ts

```diff
@@ -24,22 +24,29 @@ export const StyledWrapper = styled.div`
   backdrop-filter: blur(16px);
 `
 
-export const Modal = styled.div<{ width?: string, borderRadius?: number }>`
+export const Modal = styled.div<{
+  width?: string,
+  borderRadius?: number,
+  fullHeight?: boolean
+}>`
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   min-width: ${p => p.width ? p.width : '580px'};
   max-width: ${p => p.width ? p.width : '580px'};
   max-height: 90vh;
+  height: ${(p) => p.fullHeight ? '90vh' : 'unset'};
```

Comment:
Let's use a more versitle prop

```suggestion
  height: ${(p) => p.height ?? 'unset'};
```

Reply:
Updated!